packages/plugin-init/sources/commands/init.ts

```diff
@@ -57,17 +57,16 @@ export default class InitCommand extends BaseCommand {
   usev2 = Option.Boolean(`-2`, false, {hidden: true});
   yes = Option.Boolean(`-y,--yes`, {hidden: true});
 
-  // Used internally to avoid issues when using `-i`
+  // Deprecated; doesn't have any effect anymore, but we can't removing it for
+  // some time as it has some risks of breaking a few special setups. Probably
+  // good for removal in 4.x.
```

Comment:
Can you turn this into a `TODO` so that it comes up when searching for `TODO`s in the codebase?